requirements.txt

```diff
@@ -80,72 +74,63 @@ drf-amsterdam==0.4.1
     # via -r requirements.in
 drf-extensions==0.7.1
     # via drf-amsterdam
-drf-yasg==1.21.5
+drf-yasg==1.21.6
     # via -r requirements.in
 et-xmlfile==1.1.0
     # via openpyxl
 idna==3.4
     # via requests
 inflection==0.5.1
     # via drf-yasg
-itypes==1.2.0
-    # via coreapi
-jinja2==3.1.2
-    # via coreschema
```

Comment:
Zijn deze dependencies voor ons obsolete geworden? Met andere woorden, gebruiken we deze niet meer?